README.md

```diff
@@ -6,48 +6,48 @@ goldmark
 
 Motivation
 ----------------------
-I need a markdown parser for Go that meets following conditions:
+I need a Markdown parser for Go that meets following conditions:
 
 - Easy to extend.
-    - Markdown is poor in document expressions compared with other light markup languages like restructuredText.
-    - We have extended a markdown syntax. i.e. : PHPMarkdownExtra, Github Flavored Markdown.
+    - Markdown is poor in document expressions compared with other light markup languages like reStructuredText.
+    - We have extensions to the Markdown syntax, e.g. PHP Markdown Extra, GitHub Flavored Markdown.
 - Standard compliant.
     - Markdown has many dialects.
-    - Github Flavored Markdown is widely used and it is based on CommonMark aside from whether CommonMark is good specification or not.
+    - GitHub Flavored Markdown is widely used and it is based on CommonMark aside from whether CommonMark is good specification or not.
         - CommonMark is too complicated and hard to implement.
 - Well structured.
     - AST based, and preserves source position of nodes.
 - Written in pure Go.
 
-[golang-commonmark](https://gitlab.com/golang-commonmark/markdown) may be a good choice, but it seems copy of the [markdown-it](https://github.com/markdown-it) .
+[golang-commonmark](https://gitlab.com/golang-commonmark/markdown) may be a good choice, but it seems to be a copy of the [markdown-it](https://github.com/markdown-it).
 
-[blackfriday.v2](https://github.com/russross/blackfriday/tree/v2) is a fast and widely used implementation, but it is not CommonMark compliant and can not be extended from outside of the package since it's AST is not interfaces but structs. 
+[blackfriday.v2](https://github.com/russross/blackfriday/tree/v2) is a fast and widely used implementation, but it is not CommonMark compliant and cannot be extended from outside of the package since its AST uses not interfaces but structs.
 
-Furthermore, its behavior differs with other implementations in some cases especially of lists.  ([Deep nested lists don't output correctly #329](https://github.com/russross/blackfriday/issues/329), [List block cannot have a second line #244](https://github.com/russross/blackfriday/issues/244), etc).
+Furthermore, its behavior differs from other implementations in some cases especially of lists.  ([Deep nested lists don't output correctly #329](https://github.com/russross/blackfriday/issues/329), [List block cannot have a second line #244](https://github.com/russross/blackfriday/issues/244), etc).
 
-This behavior sometimes causes problems. If you migrate your markdown text to blackfriday based wikis from Github, many lists will immediately be broken.
+This behavior sometimes causes problems. If you migrate your Markdown text to blackfriday-based wikis from GitHub, many lists will immediately be broken.
 
 As mentioned above, CommonMark is too complicated and hard to implement, So Markdown parsers based on CommonMark barely exist.
```

Comment:
uncapitalize So

Reply:
Hi @aletson, thank you for all your great suggestions!
I'm sorry I didn't see your reviews until now, i.e. after this PR has been merged.
Let's make another pull request to incorporate your copyediting/proofreading!  :-)

Reply:
Sure. I was out of town the last few days, I will get to this shortly.